src/rest.js

```diff
@@ -1,20 +1,30 @@
 import axios_ from 'axios';
 import cookies from 'js-cookie';
 import { stringify } from 'qs';
+import NotificationBus from './utils/notifications';
```

Comment:
One of the downsides to this architecture is that it makes the REST helper module depend on Vue, even if downstreams don't need the notification bus support. We may want to consider alternative dependency relationships between the two modules.